### PR TITLE
Fixed dropdowns in RTL mode

### LINKED ISSUE #333: Language dropdowns bug in RTL locales
The language dropdowns in translate mode (both target and input) don't span into multiple columns on smaller screens when using RTL locale. This seems to be a JS bug.

## Expected behaviour:

Span correctly into multiple columns like on eg. English locale

apertium bug ltr

## Actual behaviour:

All languages go into one or two columns which are often longer than the viewport height

apertium bug rtl

### PR DESCRIPTION
**Pull request by diogoscf**
Fixes #333 

What was happening was that the available width was being calculated assuming that the source buttons were on the left and the target buttons were on the right, which is only the case if the language is ltr, so the code thought that there was much less space than there really was.

### CODE REVIEW

**Comment by diogoscf** [comment]
Umm... I don't understand what circleci is complaining about now...

File: assets/js/translator.js [hunk 1]
```diff
@@ -554,10 +554,19 @@ function populateTranslationList() {
 
     var minColumnWidth = TRANSLATION_LIST_MAX_WIDTH / TRANSLATION_LIST_MAX_COLUMNS;
 
+    var maxSrcLangsWidth, maxDstLangsWidth;
+
     // figure out how much space is actually available for the columns
-    var maxSrcLangsWidth = $(window).width() - $('#srcLanguagesDropdownTrigger').offset().left - TRANSLATION_LISTS_BUFFER,
+    if(langDirection(locale) === 'ltr') {
```
Comment (sushain97) [inline assets/js/translator.js]: This issue is that Flow knows that `locale` might not be initialized by the time this function gets called.

Comment (diogoscf) [reply assets/js/translator.js]: Makes sense. Though how would I fix that?

Comment (sushain97) [reply assets/js/translator.js]: Does this code only get called after the `locale` gets initialized? If so, we should just throw here if it's `undefined` and Flow will use that as signal for its analysis.

**Comment by sushain97** [comment]
> What was happening was that the available width was being calculated assuming that the source buttons were on the left and the target buttons were on the right, which is only the case if the language is ltr, so the code thought that there was much less space than there really was.

Ahhhh, right. Good catch. Probably my fault.

**Comment by sushain97** [comment]
BTW, I suggest `yarn install`ing locally and running Flow/ESLint through your IDE.

File: assets/js/translator.js (hunk 1 above)
Comment (diogoscf) [reply assets/js/translator.js]: I just tested and it gets called once before `locale` gets initialized and once after. But I I have thought of a way to fix this

Comment (diogoscf) [reply assets/js/translator.js]: Okay done

**Comment by jonorthwash** [comment]
> Code LGTM. @jonorthwash, I'm guessing you've tested this?

Yep, it looks really nice (i.e., the way it's supposed to) :D

**Comment by sushain97** [comment]
sorry for the extra commits, trying to get dockerhub to build commits again and it seems to have worked!

**Pull request merged**